freeipa/src/main/java/com/sequenceiq/freeipa/controller/FreeIpaV1Controller.java

```diff
@@ -194,6 +199,17 @@ public void rebootInstances(@Valid RebootInstancesRequest request) {
         rebootInstancesService.rebootInstances(accountId, request);
     }
 
+    @Override
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.ENVIRONMENT_WRITE)
+    @Retryable(value = RetryableFreeIpaClientException.class,
```

Comment:
I agree the only place that it matters is in the health checks. I would rather not push it down because otherwise, in the future I think it may lead to multilevel retries. I suspect the health service is going to be called by code that we either don't want to retry or that we don't want to retry at two levels.

Reply:
ok, I removed the retry here and also in the reboot since its basically the same situation.